digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-client-example/pom.xml

```
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>example-s3-integration-client</artifactId>
+    <name>example-s3-integration-client</name>
```

```suggestion
    <artifactId>digiwf-s3-integration-client-example</artifactId>
    <name>digiwf-s3-integration-client-example</name>
```

It would be great if the artifactId and name match the directory name